### ISSUE TITLE
最新的midway3.3.5版本下logger组件输出日志,结果没有输出

### ISSUE THREAD

**Issue by cc4dogman** [midwayjs/midway#1995]
Node Version:本地v16.13.1 阿里云v12
Midway Version: v3.3.5

项目使用https://midwayjs.org/docs/serverless/serverless_aggr创建的工程

目前启动项目本地也只能看到在src/configuration.ts中定义的logger是可以打印出来日志的,不管是appLogger或者CoreLogger方式都能打印,但Controller层之后的不管用哪种logger都无法正常输出,目前只能用console.log来临时解决,使用自定义transport注入到appLogger中也只能在系统启动阶段生效

**Comment by czy88840616** [comment]
用普通 this.ctx.logger 就行了吧。

**Comment by cc4dogman** [comment]
this.ctx.logger一样输出看不到,本地debug确实是到了Winston里面了,但就是不输出

**Comment by cc4dogman** [comment]
debug到里面去发现MidwayContextLogger里面的transport竟然是EmptyTransport....
<img width="1048" alt="image" src="https://user-images.githubusercontent.com/3928242/169641713-f26f0ffc-8c0c-4ba4-a47c-1fd2b6723a61.png">

**Comment by czy88840616** [comment]
faas-aggr 现在只有v2版本的脚手架，v3 我们还未准备好（除非手动升级，比如这位 https://github.com/midwayjs/midway/issues/1964）

**Comment by cc4dogman** [comment]
> 

是的,一开始启动也是一样的错误,后来我另外一个服务用的单函数模式是OK的,就看了下差异,然后直接引入fass后跑代码没啥问题,我感觉我要用常规应用转serverless来搞才行T_T,V3的架构要比v2合理一些,v2遍地开花

**Comment by cc4dogman** [comment]
不行啊 使用最新版本的koa-v3,然后转serverless方式依旧没应用日志输出
<img width="1389" alt="image" src="https://user-images.githubusercontent.com/3928242/169647512-de42e23d-728b-4caf-8e5b-41ae49fa2d48.png">
难道要买服务器搞了

**Comment by czy88840616** [comment]
bootstrap要改的

**Comment by czy88840616** [comment]
服务器没输出是日志level问题吧，服务器日志level默认是warn，和标准应用一致，看日志章节。
